openstack/compute/v2/extensions/resetstate/requests.go

```diff
@@ -0,0 +1,30 @@
+package resetstate
+
+import (
+	"errors"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+//ResetStatus refers to the states usable in ResetState Action
+
+type ResetStatus string
+
+const (
+	//ResetStateActive returns the state of the server as active
+	ResetStateActive ResetStatus = "active"
+
+	//ResetStateError returns the state o the server as error
+	ResetStateError ResetStatus = "error"
+)
+
+// ResetState will initiate to reset the state of a server
+func ResetState(client *gophercloud.ServiceClient, id string, state ResetStatus) (r ResetResult) {
+	if state != ResetStateActive && state != ResetStateError {
```

Comment:
We can remove this `if` block. There's no way someone can pass any other values besides `StateActive` and `StateError` now, so we don't need to check for an error :)